src/redux/categories/categories.js

```diff
@@ -0,0 +1,14 @@
+const CHECK_STATUS = 'bookstore/src/redux/categories/CHECK_STATUS';
```

Comment:
- [ ] Same issue here. Kindly write this action type value using the correct format 👍